scss/_settings_spacing.scss

```diff
@@ -1,3 +1,6 @@
+// Density multiplier
+$multi: 2 !default;
```

Comment:
This is an unclear variable name

Reply:
Also looks unused

Reply:
@anthonydillon This is not new - the density multiplier got introduced with v 1.7. I just moved it to the top of the file as it is the most important setting in this file.

Reply:
Happy to rename but it will be a breaking change so I suggest doing that with v 3.0

Reply:
I couldn't find documentation for is use. I guess that is something I missed out should be added in a follow up.

Reply:
@anthonydillon True, I've made an [issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/2656) to document it.